### PR TITLE
copy of the admission project

### PR DESCRIPTION
**Pull request by tugsem**
This branch created in order to practise  how to make code review.
I use the copy of my prior project from the admission trials.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,5 @@
-# code-review
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```
Comment (tugsem) [inline README.md:5]: Please use the readme template for your readme file.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <div id="main">
+      <h1 id="title">Tribute to Mahatma Gandhi</h1>
+    </div>
+    <div id="img-div">
+      <p id="img-caption">He was the father of the Nation India</p>
+      <img 
+        id="image"  
+        src="https://upload.wikimedia.org/wikipedia/commons/7/7a/Mahatma-Gandhi%2C_studio%2C_1931.jpg"
+      >
+    </div>
```
Comment (tugsem) [inline index.html:19]: Kindly use <header>  to wrap up your title and image.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
```
Comment (tugsem) [inline index.html:5]: Please give your page a proper title.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+} */
+
+#img-div{
+    text-align: center;
+    display: block;
+    filter: grayscale(100%);
+    width: 100%;
+    height: 100%;
+    justify-content: center;
+}
```
Comment (tugsem) [inline css/style.css:25]: Please add display: flex property to arrange items inside your header tag.

**Review by tugsem — COMMENTED** [review]
#Changes Required ♻️ 

Please check my comments below and make all required changes.

Submit for re-review when you are done. 

Thanks 😄

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+/* 
+#img-div{
+
+    text-align: center;
+}
+img{
+    width: 200px;
+    height: 150px;
+} */
```
[the remark this answers is not in the document]
Comment (ShadyShawkat) [reply css/style.css:16]: Nice suggestion 👏🏻

**Review by ShadyShawkat — APPROVED** [review]
## Hi @tugsem, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

## To highlight:
- You used GitHub features correctly. ✔️ 
- Good job creating your suggestions to be more like a suggestion and less of an order.✔️ 
- You covered all of the mistakes. ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**